libs/pipingsidesheet/src/lib/ui-sidesheet/ElectricalNetworkTab.tsx

```diff
@@ -0,0 +1,30 @@
+import { CircuitDiagram, ElectricalNetwork } from '@cc-components/shared';
+import { StyledContentWrapper } from '@cc-components/sharedcomponents';
+
+type ElectricalNetworkTab = {
+  networks: ElectricalNetwork[] | undefined;
+  itemNo: string;
+  isFetching: boolean;
+  error: Error | null;
+};
+
+export const ElecticalNetworkTab = (props: ElectricalNetworkTab): JSX.Element => {
+  const { networks, isFetching, itemNo } = props;
+
+  const diagrams = networks?.map((network) => (
```

Comment:
Weird code flow in this component?

Reply:
Moved the guard-clause to the top, and removed null-check.
Ok now?